server/models/services/CollectionJobService.js

```diff
@@ -518,24 +522,28 @@ const scheduleCollectionJob = async (
             status: COLLECTION_JOB_STATUS.QUEUED,
             testPlanReportId
         },
-        t
+        transaction
     });
 
-    return triggerWorkflow(job, testIds ?? tests.map(t => t.id), { t });
+    return triggerWorkflow(job, testIds ?? tests.map(test => test.id), {
+        transaction
+    });
 };
 
 /**
- * Gets one CollectionJob and optionally updates it, or creates it if it doesn't exist.
+ * Gets one CollectionJob and optionally updates it, or creates it if it doesn'transaction exist.
```

Comment:
A typo was created here.

Reply:
Done, nice catch finding this in the voluminous changes 😅